### PR TITLE
Added "What other think about PHP Insights" page on the website

### PR DESCRIPTION
**Pull request by krisnaw**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Documentation | yes
| Fixed tickets | #161 

Added "What others think about PHP Insights" below the `get started` section
FYI, I need to make additional components to make the Twitter widgets.js working 😄

### CODE REVIEW

**Comment by Jibbarth** [comment]
For record, I just build it to see how it looks. Great Job @krisnaw 

FireShot Capture 058 - What others think about PHP Insights - PHP Insights - localhost

File: docs/.vuepress/components/Testimonial.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template>
+    <div>
+        <h1></h1>
```
Comment (Jibbarth) [inline docs/.vuepress/components/Testimonial.vue]: If it is for more space, we should add some margin on the div above instead, WDYT ?

**Comment by krisnaw** [comment]
Hi @Jibbarth & @olivernybroe thanks for the feedback
Let me make some changes

**Comment by nunomaduro** [comment]
@krisnaw Great job! Can't wait to get this merged.

**Comment by krisnaw** [comment]
Since the layout width is so small (750px), my solution is changing the width on that page only, so it fit for 2 columns.

**Comment by nunomaduro** [comment]
Thank you for this! Just made a tweet about this: https://twitter.com/enunomaduro/status/1136952879531679745.

**Comment by olivernybroe** [comment]
Awesome work @krisnaw! 

Here is a rendered edition of it for reference :) 
localhost_8080_testimonial html

**Pull request merged**